requirements.txt

```diff
@@ -7,3 +7,4 @@ more-itertools
 transformers>=4.19.0
 ffmpeg-python==0.2.0
 pyannote.audio
+openai-whisper
```

Comment:
I would recommend pinning version or commit as they tend to introduce braking changes